sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -2504,8 +2504,12 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
   }
 
   def defaultValuesMayNotContainSubQueryExpressions(): Throwable = {
```

Comment:
Shall we change the method call of this method as `defaultValuesMayNotContain("subquery expressions")` for consistency?

Reply:
Sure, this cleans up the code. This is done.